### ISSUE TITLE
Coward

### ISSUE THREAD

**Issue by BerkanYildiz** [Ultrapowa/ucs#2]
We ask you one thing, you run away.
Don't do your fucking coward, and **fucking answer**.

Antonio RAMICELLO is using your program, and selling it.
I totally agree, i would do the same ( i did it ).

What i don't agree is that he is abusing the community. Selling an fucking bad emulator who can't even handle 1k players, for 500 $ ( and more, the fucking price depend of the fucking customer, ofc ofc niggers ) ?

You need to do something, you must do something.
I can't count how much DMCA report i received, how much DMCA report my friends received, and how much DMCA REPORT mostly all servers received.

This is fucking abuse.

**Comment by Makita43** [comment]
http://fs5.directupload.net/images/170123/coo7nm7z.png

**Comment by BerkanYildiz** [comment]
@ClashofLights Did you know, your emulator don't remove disconnected players well ? Ahahah, but ofc, you don't know coding, you can't know it ;(

**Comment by Makita43** [comment]
@BerkanYildiz Players will be automatically dropped if they disconnect ;)

**Comment by BerkanYildiz** [comment]
If they are disconnected , yea, but if they don't disconnect well ?
SocketException, DisposedObjectException and all the whistles and bells. :DDD

**Comment by aididhaiqal** [comment]
@Berkan,Yep...UCS did not capable handling SocketException...

**Comment by FICTURE7** [comment]
Hmm, I don't understand how this has anything to do with ultra tho. :/

**Comment by jeanbmar** [comment]
Don't expect people to answer your questions when you approach them with insults.

People have contributed and now request money for their work? GPL license terms allow that.
Have you contributed and shared anything on your side?

You disagree with the pricing policy? Take the code and build your own server.
You have quality concerns about the code? You are right. Feel free to improve it.

**Comment by BerkanYildiz** [comment]
@jeanbmar Then they must **leave others servers the fuck alone**.
Sending DMCA reports, and fake lawyer mails is not acceptable.

**Comment by FICTURE7** [comment]
@BerkanYildiz I think you're missing the point mate.

**Comment by zzVertigo** [comment]
@jeanbmar He has reason to bring such insults. You ran away from the first post and if the first post wasn't hard enough the second one will be harder (Which in this case it was, Berkan came in with insults), so will the third one and forth one (Or until this situation is solved.) It only gets harder until you resolve the issue.

At this point in time we literally cannot lay a finger on the code. I have people coming to me now (People who didn't even have to do with this in the begging) telling me they have received take down request on their videos or servers. It's completely unacceptable and unfair to those who we're just trying to share their server or the Ultrapowa community in general. Obviously you need to explain things a little easier for Antonio to understand because he is clearly not seeing the clear picture. Antonio seems to have the mindset that every server owner has a stack of cash ready to pay up for a "license".

TL;DR
If I tried to make a server right now using the code posted or the code I have on my computer right now. I would probably see a DMCA take down email in the next couple of days having my server online.

@ClashofLights UCS == Shit. Stop trying to act like its the best program in the world when its not. There are many issues in it (Some of which you should already be aware of) but instead you guys continue to add more stuff that create more issues then you started with.

**Comment by ghost** [comment]
@OMGItssCrayCray Let's get one thing straight, I've mentioned this previously and will again. Explain how you expect us not to issue any DCMA reports, or in general any copyright reports, when the software is literally being leaked and redistributed against someone's wishes. 

I'm sure every person can agree, even people who know nothing about anything, that IF SOMETHING IS NOT RELEASED TO THE PUBLIC IT SHOULDN'T BE USED AT ALL FOR ANY REASON AND FOR ANY PURPOSE. I'm sure people can go through something called waiting for a public release. I don't get why this is so hard to understand.

If anyone feels offended or if this causes you to feel accused or in general not the way you want to feel, I really do apologize, it is not my intention to cause any further problems not are my intentions hurt someone.

**Comment by jeanbmar** [comment]
All,
Your DMCA stuff and leaks are none of my business. 
I'm not involved in this and I'm not interested either.
From now, I'll block users showing inappropriate behaviors.

**Comment by BerkanYildiz** [comment]
Rip, didn't knew you were that dumb JB.
Anyways, closing this "issue" since nobody here can act appropriatly.

> Nobody can kill GL 🗡 .

**Comment by BerkanYildiz** [comment]
http://www.ultrapowa.com/forum/

> Rest in pieces

**Comment by StavrosNicolaou** [comment]
@DominikMC You guys are just using the latest UCS that works with the latest coc version just to get money. You are making your servers more popular and.. your forgetting that you have to release it to PUBLIC. This is why some people use the LEAKED ONE. cuz u guys are just using it for ur selfs just to get money. You went from 0.7.2.1 into 0.7.3.0 and didn't even release 0.7.2.1. Honestly, you guys are killing the community. Also, with ur new license nobody as you said can't sell ucs with he own modifications. r u kidding me? and please don't say ur not releasing ucs cuz it has a lot of bugs. if it had a lot of bugs (which it has) you wouldn't be using it to ur servers. but ur dumb what can i say.

**Comment by aididhaiqal** [comment]
Ermm,I have rights for using ucs for myself...Since i bring the leatest coc support

**Comment by Areasras** [comment]
@StavrosHacker 
We do what we want, we release when we want. Btw servers are used to find bug which can be fixed and we don't release because we think that isn't enough good."if it had a lot of bugs (which it has)" => you are saying that it has bugs and that we don't have to say that it isn't because of bug. Uhmm . If someone wants to edit UCS, he can come to us ^^. 
We are selling UCS to big servers, which earns +500$/month It is fair for the dev to get something. If you are not happy with that, find another software that's it.

**Comment by BerkanYildiz** [comment]
For the "devs" ? AHAHAHAHHAAHHAHAHAAHAHAHAHHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAH

**Comment by Areasras** [comment]
I don't see any "devs" only a "dev"

**Comment by BerkanYildiz** [comment]
Yea, you misspelled it ahahah

**Comment by Areasras** [comment]
![](https://i.imgflip.com/1icltc.jpg)

**Comment by BerkanYildiz** [comment]
You should go sleep Jean, it's late

**Comment by huseyinwincher** [comment]
Vay amk türkler bile var!
